lib/Elevate/Blockers/Repositories.pm

```diff
@@ -104,11 +104,11 @@ sub _yum_is_stable ($self) {
 
         $errors = Cpanel::SafeRun::Errors::saferunonlyerrors(qw{/usr/bin/yum makecache});
         if ( $errors =~ m/\S/ms ) {
-            ERROR('yum appears to be unstable. Please address this before upgrading');
+            ERROR('yum appears to be unstable. Specifically, "yum makecache" is generating an error.  Please address this before upgrading');
```

Comment:
Lets expand this out a bit further.  Maybe something like this:

```
my $error_msg = <<~'EOS';
'/usr/bin/yum makecache' failed to return cleanly.  This could be due to a temporary mirror problem or it could be indicative of a larger issue such as a broken repository.  Since this script relies heavily on yum, this will need to be addressed before upgrading.

You may want to consider reaching out to cPanel Support for assistance:

https://docs.cpanel.net/knowledge-base/technical-support-services/how-to-open-a-technical-support-ticket/

EOS
```

Also, please execute `make` before resubbing this so that the fatpacker will update the main `elevate-cpanel` script

Reply:
Cool.  Easily done!